hierarqcal/qiskit/qiskit_helper.py

```diff
@@ -65,10 +66,57 @@ def get_circuit_qiskit(hierq, symbols=None, barriers=True):
             qiskit_default_unitary = get_qiskit_default_unitary(layer)
             layer.set_edge_mapping(qiskit_default_unitary)
         for unitary in layer.edge_mapping:
+
+            if isinstance(unitary.function, str):
+                unitary = get_circuit_from_string(unitary)
```

Comment:
Can you add a comment here that says we're updating the qunitary object's function attribute to a qiskit function based on the provided string

Reply:
yes, thanks for the note!